CHANGELOG.md

```diff
@@ -4,6 +4,57 @@
 
 <!-- next version -->
 
+## v0.9.0
+
+This release includes version 0.104.0 of the upstream Collector components.
+
+The individual upstream Collector changelogs can be found here:
+
+v0.104.0:
+
+- <https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.104.0>
+- <https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.104.0>
+
+<details>
+<summary>Highlights from the upstream Collector changelog</summary>
+</br>
+
+### 🛑 Breaking changes 🛑
+
+This release includes 2 very important breaking changes.
```

Comment:
I think those should be above the `<details>` section so they're expanded by default (also in the /releases description).

Reply:
done